common/schema-builder/src/test/java/io/smallrye/graphql/index/SchemaBuilderTest.java

```diff
@@ -171,6 +171,28 @@ public void testSchemaWithBatchSourceFieldNameDuplicates() {
         }
     }
 
+    @Test
+    public void testSchemaWithInheritFieldBySubtype() {
+        Indexer indexer = new Indexer();
+        indexDirectory(indexer, "io/smallrye/graphql/index/inherit/");
+
+        IndexView index = indexer.complete();
+        Schema schema = SchemaBuilder.build(index);
+        // Get Types
+        Map<String, Type> types = schema.getTypes();
+        Map<String, Type> interfaces = schema.getInterfaces();
+        Type containerType = types.get("ContainerType");
+        Field containerTypeField = containerType.getFields().get("inheritField");
+        Type containerInterface = interfaces.get("ContainerInterface");
+        Field containerInterfaceField = containerInterface.getFields().get("inheritField");
+
+        // TODO: Should be FieldInterface according to GraphQL Spec: https://spec.graphql.org/October2021/#sec-Objects.Type-Validation
+        assertEquals("io.smallrye.graphql.index.inherit.FieldInterface",
```

Comment:
Just to make sure I understand correctly - you think this should be `FieldType` instead of `FieldInterface`?